classes/class.tapestry-node.php

```diff
@@ -99,6 +100,7 @@ public function __construct($tapestryPostId = 0, $nodeMetaId = 0)
             'lng' => '',
         ];
         $this->popup = null;
+        $this->answers = (object) [];
```

Comment:
I think that we can use new stdClass() here instead of converting an array to an object. 
https://www.php.net/manual/en/language.types.object.php